common/src/WooCommerce.php

```diff
@@ -132,19 +132,34 @@ private function sendInvite($order_id) {
         }
 
         // send invite
+        $is_invite_added = true;
         $api = new API($api_domain, $shop_id, $api_key);
         try {
             $api->invite($data);
-        } catch (WebwinkelKeurAPIAlreadySentError $e) {
-            // that's okay
-        } catch (WebwinkelKeurAPIError $e) {
-            $this->logApiError($e);
+        } catch (Exception $e) {
+            $is_invite_added = false;
+            if ($e instanceof WebwinkelKeurAPIAlreadySentError) {
```

Comment:
Why not just use try/catch as it is intended, and return after adding the error comment, so you don't need this `$is_invite_added` and all the if statements?

See this: https://github.com/webwinkelkeur/wordpress/commit/6c3a27ad62597a1088ec181fe217515c9aed2928

You can fetch & cherry pick this commit, or do it yourself if you prefer.